### DISCUSSION TITLE
How to set state init data when making initial transaction of uninit state address?

### DISCUSSION THREAD

**Discussion by autumn2sky** [trustwallet/wallet-core#4185] [Q&A] [answered]
I found a function that make state init for V4R2 wallet. - buildV4R2StateInit()
1. How to insert this init data when building transaction data?
2. Where are functions for V3R2 or V5 wallet contract?

**Comment by satoshiotomakan** [comment, marked as answer]
Hi @autumn2sky, thanks for asking!
1. `StateInit` to initialize the uninit account will be automatically set if `sequence = 0`, you don't need to set it manually
2. We don't support V3R2 at the moment, and don't plan. V5 is supported, and the `StateInit` is set automatically the same way as for V4R2
